web-components/src/utils/generateScssFromTokens.js

```diff
@@ -196,10 +196,8 @@ const generateThemeStylesheets = () => {
 
       themeStyleFiles[`${designTheme}${lightDarkTheme}`] += `@import "@/wc_scss/themes/global--${designTheme}.scss";\n`;
 
-      if (designTheme === "mdv2") {
-        themeStyleFiles[`${designTheme}${lightDarkTheme}`] +=
-          `@import "@momentum-design/tokens/dist/scss/theme/webex/${lowercaseLightDarkTheme}-stable.scss";\n`;
-      }
+      themeStyleFiles[`${designTheme}${lightDarkTheme}`] +=
+        `@import "@momentum-design/tokens/dist/scss/theme/webex/${lowercaseLightDarkTheme}-stable.scss";\n`;
 
```

Comment:
This allows the momentum-design/tokens to be accessed in all themes. This is needed when developing new components that do not have any design in other themes. in this case all themes can be designed to look the same